builder.json

```diff
@@ -74,6 +74,20 @@
                     "!imports": [ "node-16" ]
                 }
             }
+        },
+        "windows": {
+            "!build_steps": [
+                [
+                    "node",
+                    "scripts/build.js",
+                    "-DAWS_WARNINGS_ARE_ERRORS=ON"
+                ],
+                [
+                    "npm",
+                    "install",
+                    "--unsafe-perm"
+                ]
+            ]
```

Comment:
Can we update `build_steps` at line 109 to run the `node scripts/build.js -DAWS_WARNINGS_ARE_ERRORS=ON` before `npm install`?
In the original PR, Mike said it related to the arm mac, but I guess i don't know exactly why. Maybe it's not the problem now??

Try it out, and see if we have any issues. Since it makes more sense to have all platform to take the warnings as errors.

Reply:
Seems to be good. Let me know if there's anything else.